src/sketch.js

```diff
@@ -1339,12 +1339,12 @@ VectorPaintEditorMorph.prototype.populatePropertiesMenu = function () {
             function () { return myself.shift; }
             );
 
-    alignColor.add(pc.primaryColorViewer);
     alignColor.add(pc.secondaryColorViewer);
+    alignColor.add(pc.primaryColorViewer);
     alignColor.fixLayout();
 
     c.add(pc.colorpicker);
-    c.add(new TextMorph(localize('Primary color      Secondary color')));
+    c.add(new TextMorph(localize('    Edge color             Fill color')));
```

Comment:
this is the part I would love to change. Using magic spaces to align what should be two individual labels doesn't work well for translations. But that's not your change, so don't worry about it. I'll take care of turning this into separate strings/morphs when I integrate your changes. Thanks!